docs/tutorial/sandbox.md

```diff
@@ -17,7 +17,7 @@ further configuration. If you want to disable the sandbox for a process, see the
 [Disabling the sandbox for a single process](#disabling-the-sandbox-for-a-single-process)
 section.
 
-## Sandbox behaviour in Electron
+## Sandbox behavior in Electron
```

Comment:
This is not an incorrect spelling - it's British English. We can change this given the majority of our documentation follows American English but strictly speaking it's not a typo.

Reply:
I have changed the description of the PR. Thank you!